### DISCUSSION TITLE
Just buy your own chromebook!

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#189] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
If you idiotic children are willing to downgrade and even un-enroll your school chromebook just so you could block extensions, something that could get you thrown in Jail as specified by Yeetyeet, then you might as well just pay $250 for a chromebook that is all yours to use at school, I highly doubt anyone would notice if you use a mobile hotspot as your network.

**Comment by 3kh0** [comment]
Don't buy a Chromebook, by a windows PC, or even better, get Linux. But if you have a small budget yes get your own Chromebook. I saw one in Walmart for $75, 4gb ram and 64gb ssd

**Reply by ghost** [reply]
YES! I sell laptops too, but don't ship
